### PR TITLE
[DE] Add `Beatmapping/Beatmap_submission`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Translation note: Keep in mind that `Vollständige Beatmaps` is a translation in stable which differs from newer ones. The current translation used for `pending beatmaps` is `ausstehende Beatmaps`.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmapping/Beatmap_submission/de.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+![](img/bss_warning-DE.png "Bildschirm des Beatmap-Einreichungssystems")
+
+Beim Anwählen von `Beatmap hochladen ...` aus dem Dropdown `Datei` im Editor (Kürzel: `Strg` + `Shift` + `U`) öffnet sich das Fenster für das **Beatmap-Einreichungssystem** (auch **Beatmap Submission System** oder ***BSS*** genannt). Hier werden zu Anfang Informationsquellen aufgelistet, um Nutzern zu helfen, Fragen im Bezug auf Mapping zu stellen, Feedback für ihre Beatmaps zu finden und dass ihre Beatmaps für das Ranking geeignet sind. Solltest du Probleme mit dem BSS haben, dann siehe den Ratgeber über [Probleme mit dem BSS](/wiki/Guides/BSS_Issues).
+
+Wenn die Beatmap, die ein Nutzer hochlädt, nicht bereits auf der osu!-Webseite ist, dann wird das BSS-Fenster anzeigen, wie viele Neuveröffentlichungen einem Nutzer noch zustehen. Sofern die Beatmap nominiert ist, warnt das Fenster davor, dass die Nominierung durch das Aktualisieren der Beatmap zurückgesetzt wird. Wenn die Beatmap sich auf dem Friedhof befindet, dann weist das Fenster daraufhin, dass die Beatmap in die Kategorie "Ausstehend" verschoben wird.
```
Comment (FamousPig) [inline wiki/Beatmapping/Beatmap_submission/de.md:9]:
```suggestion
Wenn die Beatmap, die ein Nutzer hochlädt, nicht bereits auf der osu!-Webseite ist, dann wird das BSS-Fenster anzeigen, wie viele Neuveröffentlichungen einem Nutzer noch zustehen. Sofern die Beatmap nominiert ist, warnt das Fenster davor, dass die Nominierung durch das Aktualisieren der Beatmap zurückgesetzt wird. Wenn die Beatmap sich auf dem Friedhof befindet, dann weist das Fenster darauf hin, dass die Beatmap in die Kategorie "Ausstehend" verschoben wird.
```

File: wiki/Beatmapping/Beatmap_submission/de.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Der Abschnitt `Kommentar des Erstellers` ermöglicht Benutzereingaben, die entlang der Auflistungsseite einer Beatmap erscheinen. Das Eingabefeld unterstützt [BBCode](/wiki/BBCode)-Formatierung.
+
+Am unteren Ende des Fensters sind zwei Checkboxen erkennbar. Die erste trägt den Text `Bei Antworten Benachrichtigung per E-Mail schicken`, wodurch die Beatmap zur [Modding-Beobachtungsliste](https://osu.ppy.sh/beatmapsets/watches) des Nutzers hinzugefügt wird. Die zweite ist mit `Nach dem Einreichen im Browser öffnen` beschriftet, wodurch die Auflistungsseite der Beatmap in deinem Standardbrowser geladen wird.
+
+## Limitierungen
+
+Beatmaps werden nicht eingereicht, wenn sie die Online-Dateigröße oder das Schwierigkeitslimit überschreiten. Die Beschränkung der Dateigröße liegt bei 5 MB sowie zusätzliche 10 MB für jede Minute an Beatmap-Länge. Das Maximum beträgt 100 MB. Die Schwierigkeitsgrenze ist aktuell 128.
```
Comment (FamousPig) [inline wiki/Beatmapping/Beatmap_submission/de.md:23]:
```suggestion
Beatmaps werden nicht eingereicht, wenn sie das Online-Dateigrößen- oder Schwierigkeitslimit überschreiten. Die Beschränkung der Dateigröße liegt bei 5 MB sowie zusätzliche 10 MB für jede Minute an Beatmap-Länge. Das Maximum beträgt 100 MB. Die Schwierigkeitsgrenze ist aktuell 128.
```

File: wiki/Beatmapping/Beatmap_submission/de.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Benutzern wird es erlaubt, eine limitierte Anzahl an ausstehenden Beatmaps gleichzeitig auf der Webseite zu haben. Die Beschränkung variiert, abhängig davon, wie viele gerankte Beatmaps ein Nutzer hat und ob er gerade ein [osu!supporter](/wiki/osu!supporter) ist. Benutzern ohne osu!supporter können 4 ausstehende Beatmaps plus 1 pro gerankter Beatmap haben (bis zu 2). Mit osu!supporter wird die Grenze auf 8 ausstehende Beatmaps plus 1 pro gerankter Beatmap erhöht (bis zu 12), also insgesamt 20.
```
Comment (FamousPig) [inline wiki/Beatmapping/Beatmap_submission/de.md:25]:
```suggestion
Benutzern wird es erlaubt, eine limitierte Anzahl an ausstehenden Beatmaps gleichzeitig auf der Webseite zu haben. Die Beschränkung variiert, abhängig davon, wie viele gerankte Beatmaps ein Nutzer hat und ob er gerade ein [osu!supporter](/wiki/osu!supporter) ist. Benutzer ohne osu!supporter können 4 ausstehende Beatmaps plus 1 pro gerankter Beatmap haben (bis zu 2). Mit osu!supporter wird die Grenze auf 8 ausstehende Beatmaps plus 1 pro gerankter Beatmap erhöht (bis zu 12), also insgesamt 20.
```

File: wiki/Beatmapping/Beatmap_submission/de.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Die Upload-Geschwindigkeit unterscheidet sich, abhängig davon, welche Dateien geändert wurden. Sofern nur die `.osu`-Dateien verändert wurden, sind sie die einzigen Dateien im Beatmap-Ordner, die verarbeitet und hochgeladen werden. Wenn andere Dateien geändert werden, werden alle Dateien im Beatmap-Ordner verarbeitet und hochgeladen.
```
Comment (FamousPig) [inline wiki/Beatmapping/Beatmap_submission/de.md:27]:
```suggestion
Die Upload-Geschwindigkeit unterscheidet sich abhängig davon, welche Dateien geändert wurden. Sofern nur die `.osu`-Dateien verändert wurden, sind sie die einzigen Dateien im Beatmap-Ordner, die verarbeitet und hochgeladen werden. Wenn andere Dateien geändert werden, werden alle Dateien im Beatmap-Ordner verarbeitet und hochgeladen.
```

File: wiki/Beatmapping/Beatmap_submission/de.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Einreichen
+
+[Beatmaps](/wiki/Beatmap) können auf die osu!-Webseite über den [Editor im Spiel](/wiki/Client/Beatmap_editor) eingereicht werden. Durch das Einreichen von Beatmaps können andere Nutzer ein Augenmerk auf diese werfen und die Beatmaps möglicherweise in die Kategorien [Ranked](/wiki/Beatmap/Category#ranked) oder [Loved](/wiki/Beatmap/Category#loved) eintreten.
```
Comment (FamousPig) [inline wiki/Beatmapping/Beatmap_submission/de.md:3]:
```suggestion
[Beatmaps](/wiki/Beatmap) können auf die osu!-Webseite über den [Editor im Spiel](/wiki/Client/Beatmap_editor) eingereicht werden. Durch das Einreichen von Beatmaps können andere Nutzer einen Blick auf diese werfen und die Beatmaps möglicherweise in die Kategorien [Ranked](/wiki/Beatmap/Category#ranked) oder [Loved](/wiki/Beatmap/Category#loved) eintreten.
```
I don't believe `Ein Augenmerk auf etwas werfen` is a real idiom. The only one I was able to find was `sein Augenmerk auf etwas richten`, which would be focusing on a particular detail and therefore not really applicable here.

File: wiki/Beatmapping/Beatmap_submission/de.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+Wenn die Beatmap, die ein Nutzer hochlädt, nicht bereits auf der osu!-Webseite ist, dann wird das BSS-Fenster anzeigen, wie viele Neuveröffentlichungen einem Nutzer noch zustehen. Sofern die Beatmap nominiert ist, warnt das Fenster davor, dass die Nominierung durch das Aktualisieren der Beatmap zurückgesetzt wird. Wenn die Beatmap sich auf dem Friedhof befindet, dann weist das Fenster daraufhin, dass die Beatmap in die Kategorie "Ausstehend" verschoben wird.
+
+## Einreichungsoptionen
+
+![](img/bss_submitting-DE.png "Upload-Bildschirm des Beatmap-Einreichungssystems")
+
+Beim Anklicken des Buttons `NEUE Beatmap hochladen!` oder `Beatmap aktualisieren!` hat ein Nutzer die Wahl, ihre Beatmap in die Kategorien `Beatmaps in Arbeit` oder `Vollständige Beatmaps` hochzuladen. WIP-Beatmaps können nicht für das Ranking nominiert werden, während ausstehende Beatmaps es können.
```
Comment (FamousPig) [inline wiki/Beatmapping/Beatmap_submission/de.md:15]: Just want to confirm if the use of `ihre` here is intentional.

File: wiki/Beatmapping/Beatmap_submission/de.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Einreichen
+
+[Beatmaps](/wiki/Beatmap) können auf die osu!-Webseite über den [Editor im Spiel](/wiki/Client/Beatmap_editor) eingereicht werden. Durch das Einreichen von Beatmaps können andere Nutzer ein Augenmerk auf diese werfen und die Beatmaps möglicherweise in die Kategorien [Ranked](/wiki/Beatmap/Category#ranked) oder [Loved](/wiki/Beatmap/Category#loved) eintreten.
+
+![](img/bss_warning-DE.png "Bildschirm des Beatmap-Einreichungssystems")
+
+Beim Anwählen von `Beatmap hochladen ...` aus dem Dropdown `Datei` im Editor (Kürzel: `Strg` + `Shift` + `U`) öffnet sich das Fenster für das **Beatmap-Einreichungssystem** (auch **Beatmap Submission System** oder ***BSS*** genannt). Hier werden zu Anfang Informationsquellen aufgelistet, um Nutzern zu helfen, Fragen im Bezug auf Mapping zu stellen, Feedback für ihre Beatmaps zu finden und dass ihre Beatmaps für das Ranking geeignet sind. Solltest du Probleme mit dem BSS haben, dann siehe den Ratgeber über [Probleme mit dem BSS](/wiki/Guides/BSS_Issues).
```
Comment (FamousPig) [inline wiki/Beatmapping/Beatmap_submission/de.md:7]:
```suggestion
Beim Anwählen von `Beatmap hochladen ...` aus dem Dropdown `Datei` im Editor (Kürzel: `Strg` + `Shift` + `U`) öffnet sich das Fenster für das **Beatmap-Einreichungssystem** (auch **Beatmap Submission System** oder ***BSS*** genannt). Hier werden zu Anfang Informationsquellen aufgelistet, um Nutzern zu helfen, Fragen im Bezug auf Mapping zu stellen, Feedback für ihre Beatmaps zu finden und sicherzustellen, dass ihre Beatmaps für das Ranking geeignet sind. Solltest du Probleme mit dem BSS haben, dann siehe den Ratgeber über [Probleme mit dem BSS](/wiki/Guides/BSS_Issues).
```

**Review by FamousPig — CHANGES_REQUESTED** [review]

File: wiki/Beatmapping/Beatmap_submission/de.md (hunk 6 above)
Comment (The-Last-Cookie) [reply wiki/Beatmapping/Beatmap_submission/de.md:15]: No, it's not. Thanks for spotting this.

```suggestion
Beim Anklicken des Buttons `NEUE Beatmap hochladen!` oder `Beatmap aktualisieren!` hat ein Nutzer die Wahl, seine Beatmap in die Kategorien `Beatmaps in Arbeit` oder `Vollständige Beatmaps` hochzuladen. WIP-Beatmaps können nicht für das Ranking nominiert werden, während ausstehende Beatmaps es können.
```

**Review by FamousPig — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**